### PR TITLE
Fix scribing scaling

### PR DESCRIPTION
**Pull request by kxmbrian**
Fix for #336. Scribbles should now display consistently on different browser / screen / canvas sizes.

Other minor improvements also included.

### CODE REVIEW

**Comment by wangqiang1208** [comment]
@kxmbrian I occasionally found this: https://github.com/Coursemology/coursemology.org/blob/development/app/models/assessment/submission.rb#L23

I think it's a typo ?

**Comment by kxmbrian** [comment]
Good catch. It seems like neither general_answers or scribing_answers are invoked anywhere. Should I be commenting / removing the unnecessary code actually?

**Comment by wangqiang1208** [comment]
Yah. seems no where called general_answers so nothing was broken.  But better fix it in case someone called it in future.

**Comment by kxmbrian** [comment]
@wangqiang1208, this feature is currently frozen and ready for a final check before merge. Could you help to review?

**Comment by jsyeo** [comment]
@wangqiang1208, good to merge?

**Comment by wangqiang1208** [comment]
@kxmbrian @jsyeo  Give me a while, I'm testing this !

File: app/views/assessment/mission_submissions/_scribing_canvas.html.erb [hunk 1]
```diff
@@ -67,8 +63,10 @@
 					</a>
 				</div>
 				<div class="btn-group hidden" id="scribing-drawing-tools-<%= qid %>">
-          <input type="text" class="scribing-color-val" id="scribing-color-<%= qid %>"></input>
-			    <input type="range" id="scribing-width-<%= qid %>" value="2" min="0" max="30" style="width: 100px"></input>
+					<div class="panel">
+	          <input type="text" class="scribing-color-val" id="scribing-color-<%= qid %>"></input>
```
Comment (wangqiang1208) [inline app/views/assessment/mission_submissions/_scribing_canvas.html.erb]: Not aligned here.

**Comment by wangqiang1208** [comment]
@kxmbrian @jsyeo  I've tested this PR, Tests on Safari and Chrome shows that #336 is fixed.

I haven't looked at javascript code. And this PR seems only changed  the scribing related stuff, so should not have side effects on other assessment parts.

Except some minor issues, I think this is good to merge.

**Comment by jsyeo** [comment]
@wangqiang1208 minor issues?

**Comment by wangqiang1208** [comment]
@jsyeo Sorry, didn't notice your message, this can be merged.

**Comment by jsyeo** [comment]
> Except some minor issues, I think this is good to merge. 

What are the minor issues?

**Comment by wangqiang1208** [comment]
See my comments, actually just line alignments.

**Comment by jsyeo** [comment]
> See my comments

Which one?

**Comment by wangqiang1208** [comment]
Cannot find the last outdated one ?

File: app/views/assessment/mission_submissions/_scribing_canvas.html.erb (hunk 1 above)
Comment (wangqiang1208) [inline app/views/assessment/mission_submissions/_scribing_canvas.html.erb]: @jsyeo it was here

**Comment by jsyeo** [comment]
@kxmbrian please split your PR next time. You are trying to fix 2-3 issues here. A bit hard for me and @wangqiang1208 to review. Thanks. :smile:

**Comment by kxmbrian** [comment]
@jsyeo What should I do if one PR is built on a previous PR that has not been merged?

**Comment by jsyeo** [comment]
@kxmbrian  just have to let us know that the PR depends on another PR. (e.g. https://github.com/Coursemology/coursemology2/pull/78#issue-55006470) I will merge the one that is depended on first.

**Comment by kxmbrian** [comment]
ok thanks :+1:

**Pull request merged**